src/scss/base/_base.scss

```diff
@@ -1,48 +1,4 @@
 :root {
-  /* Manifold colors */
```

Comment:
@zdavis After adding the GlobalStyles from styled-components, I removed all of the repeat CSS variable names and utility classes from the SASS files